rules/xcodeproj.bzl

```diff
@@ -911,6 +938,8 @@ https://www.rubydoc.info/github/CocoaPods/Xcodeproj/Xcodeproj/Constants
         "additional_files": attr.label_list(allow_files = True, allow_empty = True, default = [], mandatory = False),
         "additional_prebuild_script": attr.string(default = "", mandatory = False),  # Note this script will run BEFORE Bazel build script
```

Comment:
We should remove this no?

Reply:
mine is different, this is added at 
<img width="593" alt="Screen Shot 2021-06-28 at 4 06 16 PM" src="https://user-images.githubusercontent.com/493722/123697324-c62d0f80-d82a-11eb-9cae-5a938adfa296.png">

mine is at:
<img width="175" alt="Screen Shot 2021-06-28 at 4 06 36 PM" src="https://user-images.githubusercontent.com/493722/123697356-d1803b00-d82a-11eb-942d-e94020e26596.png">

Reply:
> mine is different, this is added at
> <img alt="Screen Shot 2021-06-28 at 4 06 16 PM" width="593" src="https://user-images.githubusercontent.com/493722/123697324-c62d0f80-d82a-11eb-9cae-5a938adfa296.png">
> 
> mine is at:
> <img alt="Screen Shot 2021-06-28 at 4 06 36 PM" width="175" src="https://user-images.githubusercontent.com/493722/123697356-d1803b00-d82a-11eb-942d-e94020e26596.png">

So I think that we should potentially remove that build phase, I know I added it but I am unsure if that is the right "Bazel" way to do this. I believe developers should really only be using pre/post actions in a Bazel Build, offering this build phase gives the wrong impression about what is going on with this project.

This is just my opinion, trying to reduce visible levers for developers when things go wrong

Reply:
Sounds good. I was trying to understand if others were still using this code path. 

On another note, how are you handling surfacing errors in Xcode from these scripts? I know how to accomplish this in build phases but not in scheme pre/post actions

Reply:
so right now the only way I can think of is to write to a log to surface errors. Therefore I would be very cautious about when to use it and use pre_build script phase whenever possible.